packages/connect-popup/src/log.tsx

```diff
@@ -101,14 +101,10 @@ const logInConsole = (logs: any[]) => {
 const useLogWorker = (setLogs: React.Dispatch<React.SetStateAction<any[]>>) => {
     let logWorker: SharedWorker | undefined;
 
-    if (SharedWorker) {
-        try {
-            logWorker = new SharedWorker('./workers/shared-logger-worker.js');
-        } catch (error) {
-            console.warn('Failed to initialize SharedWorker');
-        }
-    } else {
-        console.warn('SharedWorker is not supported in this browser');
+    try {
+        logWorker = new SharedWorker('./workers/shared-logger-worker.js');
```

Comment:
wouldn't this still be initiating logger worker in popup even when it is set to false in init? 
my feeling is yes. and it could be taken care of - connect settings arrives to popup in handshake message I think.

Reply:
Yes, it will.
My intention was not to stop it everywhere but only in 3rd party side. Of course we could stop it everywhere.

Reply:
I made the changes so shared logger is not loaded in popup when connect settings set it to `false` but not in the `log.html` page `src/logs.tsx` because that page it is actually not getting the connect settings and it is kind of independent of popup when popup is not loading the shared-worker. Besides there is no point of having the log page without the shared-worker for logs.